modules/EvseV2G/sdp.cpp

```diff
@@ -220,15 +220,6 @@ int sdp_init(struct v2g_context* v2g_ctx) {
         return -1;
     }
 
-    if (v2g_ctx->local_tcp_addr) {
-        memcpy(&sdp_addr.sin6_addr, &v2g_ctx->local_tcp_addr->sin6_addr, sizeof(sdp_addr.sin6_addr));
-    } else if (v2g_ctx->local_tls_addr) {
-        memcpy(&sdp_addr.sin6_addr, &v2g_ctx->local_tls_addr->sin6_addr, sizeof(sdp_addr.sin6_addr));
-    } else {
-        dlog(DLOG_LEVEL_ERROR, "Neither TLS nor TCP server on %s", v2g_ctx->if_name);
-        return -1;
-    }
-
```

Comment:
I'm guessing that this (binding to a specific IP address) probably did not work, because the value of `sdp_addr.sin6_addr` was overwritten with `in6addr_any` 15 lines below:
https://github.com/EVerest/everest-core/blob/b5b315d3b56304030df45752f9b1ee9137fde662/modules/EvseV2G/sdp.cpp#L236

Reply:
Thanks for the explanation. Sound reasonable. Probably the inaddr_any was added because the multicast didn't work in the first place...

I agree with the conclusions, so LGTM.